code/addons/a11y/src/components/A11yContext.tsx

```diff
@@ -215,16 +215,18 @@ export const A11yContextProvider: FC<PropsWithChildren> = (props) => {
   const handleSelect = useCallback(
     (itemId: string, details: ClickEventDetails) => {
       const [type, id] = itemId.split('.');
-      const index =
-        results?.[type as RuleType]
-          ?.find((r) => r.id === id)
-          ?.nodes.findIndex((n) => details.selectors.some((s) => s === String(n.target))) ?? -1;
-      if (index !== -1) {
-        const key = `${type}.${id}.${index + 1}`;
-        setSelectedItems(new Map([[`${type}.${id}`, key]]));
-        setTimeout(() => {
-          document.getElementById(key)?.scrollIntoView({ behavior: 'smooth', block: 'center' });
-        }, 100);
+      const { helpUrl, nodes } = results?.[type as RuleType]?.find((r) => r.id === id) || {};
+      const openedWindow = helpUrl && window.open(helpUrl, '_blank', 'noopener,noreferrer');
```

Comment:
isn't it bad for a11y if the link opens in a new tab?

Reply:
Maybe, I don't know. UX wise at least we have an icon to indicate this behavior. For screenreaders I have no idea whether this is okay or not. Either way I think as a general rule we should always be opening 3rd party links in a new tab.